src/css/fluid-covid-map-viz.css

```diff
@@ -379,3 +452,95 @@ body::after {
 .fl-mapviz-filter-wait .fl-mapviz-filter-icon {
     background-image: url(../img/Wait.svg);
 }
+
+@media screen and (min-width: 1024px) {
+    /******** The main container holding all 3 panels: map, a11y feature list and search results ********/
+
+    /* Start: arrange positions of sub containers */
+    .fl-mapviz-container {
+        display: grid;
+        grid-template-columns: 1fr 3fr auto;
+        height: 42rem;
+    }
+
+    .fl-mapviz-control-wrapper {
+        grid-column-end: 3;
+        grid-column-start: 1;
+        grid-row-end: 2;
+        grid-row-start: 1;
+        height: 5rem;
+        margin: 0.5rem 2rem;
+        position: relative;
+        width: 100%;
+    }
+
+    .fl-mapviz-filter-panel {
+        display: block;
+        grid-column-end: 4;
+        grid-column-start: 3;
+        grid-row-end: 3;
+        grid-row-start: 1;
+    }
+
+    .fl-mapviz-city-list,
+    .fl-mapviz-search-results {
+        border-radius: 0;
+        grid-column-end: 2;
+        grid-column-start: 1;
+        grid-row-end: 3;
+        grid-row-start: 2;
+        height: inherit;
+        position: inherit;
+    }
+
+    .fl-mapviz-map {
+        grid-column-end: 3;
+        grid-column-start: 2;
+        grid-row-end: 3;
+        grid-row-start: 2;
+        height: auto;
+    }
+
+    /* End: arrange positions of sub containers */
+
+    /******** The panel with the query label, the query input box and the filter control button **********/
+    .fl-mapviz-control-holder {
+        width: 90%;
+    }
+
+    /******** The panel of the a11y feature filter list **********/
+    .fl-mapviz-query-label {
+        display: block;
+        margin-bottom: 1rem;
+    }
+
+    /******** Expand/collapse button for the city list and the search results panels ********/
+    .fl-mapviz-expand-collapse-button-holder {
+        display: none;
+    }
+
+    .fl-mapviz-expand-collapse-button-holder[aria-expanded="false"] ~ .fl-mapviz-search-result-list {
+        display: block;
+    }
+
+    /******** The panel for displaying cities ********/
+    .fl-mapviz-city-list-title {
+        align-self: auto;
+    }
+
+    /******** The panel for displaying search results ********/
+    .fl-mapviz-search-results {
+        height: 36rem;
+        margin-right: 1rem;
+    }
+
+    .fl-mapviz-back-button {
+        justify-content: left;
+        padding-left: 1rem;
```

Comment:
I found that this padding definition is causing a misalignment of the entire button block - together with the "width: 100%" directive of the container, it causes the back button to stick out by 1 rem into the map